### PR TITLE
Update Release Collateral for Stacks

### PR DESCRIPTION
**Pull request by bd-dean**
modified three files: 
README.md
stacks/dlrs/README.md
stacks/dlrs/releasenotes.md

### CODE REVIEW

File: stacks/dlrs/releasenote.md [hunk 1]
```diff
@@ -43,6 +63,10 @@ The official Deep Learning Reference Stack Docker images are hosted at: https://
 
  * Pull from the [Eigen version](https://hub.docker.com/r/clearlinux/stacks-dlrs-oss/)
  * Pull from the [Intel MKL-DNN version](https://hub.docker.com/r/clearlinux/stacks-dlrs-mkl/)
+ * Pull from the [PyTorch version](https://hub.docker.com/r/clearlinux/stacks...)
+ * Pull from the [PyTorch with Intel MKL version](https://hub.docker.com/r/clearlinux/stacks...)
+ * Pull from the [PyTorch with Intel MKL-DNN version](https://hub.docker.com/r/clearlinux/stacks...)
```
Comment (bryteise) [inline stacks/dlrs/releasenote.md]: Just being noisy but are those links valid?

Comment (bd-dean) [reply stacks/dlrs/releasenote.md]: They need review before this can be merged.

Comment (rahulunair) [reply stacks/dlrs/releasenote.md]: +1 looks good to me, will provide you the new links asap.

Comment (bd-dean) [reply stacks/dlrs/releasenote.md]: Links updated, ready to go.

**Comment by bryteise** [comment]
LGTM, +1

**Comment by rahulunair** [comment]
@bryteise @ganeshmaharaj  could one of you help in merging this... ?

**Comment by ganeshmaharaj** [comment]
@rahulunair unfortunately i do not have write access to this repository.

**Comment by bryteise** [comment]
I'll use power wisely.

**Pull request merged**